frontend/public/components/operator-hub/operator-hub-items.tsx

```diff
@@ -335,12 +336,14 @@ export const OperatorHubTileView = requireOperatorGroup(
       const { uid, name, imgUrl, iconClass, provider, description, installed } = item;
       const normalizedIconClass = iconClass && `icon ${normalizeIconClass(iconClass)}`;
       const vendor = provider ? `provided by ${provider}` : null;
+      const tileImgUrl = imgUrl || catalogImg;
+
       return (
         <CatalogTile
           id={uid}
           key={uid}
           title={name}
-          iconImg={imgUrl}
+          iconImg={tileImgUrl}
```

Comment:
In `frontend/public/components/catalog/catalog-items.jsx` you switch `titleImgUrl` to `imgUrl`.  Should be consistent in both places.